pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -2471,3 +2471,18 @@ def test_dt64arr_addsub_object_dtype_2d():
     assert result2.shape == (4, 1)
     assert result2.freq is None
     assert (result2.asi8 == 0).all()
+
+
+def test_sum_empty_df_series():
+    # Calling the following defined sum function returned an error for dataframes but
+    # returned NaT for series. # Check that the API is consistent in this sense when
+    # operating on empty Series/DataFrames. See GH:33704 for more information
```

Comment:
frame tests go here (you can rename slightly); these should eventually be moved into test_reductions.py

class TestDataFrameReductions:
    def test_min_max_dt64_with_NaT

series are here:
tests/series/test_reductions.py

in tests/frame/test_analytics.py

Reply:
Though I could not find the class you mentioned

> pls add a whatsnew note as well, 1.1 bug fixes datetimelike section

Done.

Reply:
done